server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/DeletedMailsDAO.java

```diff
@@ -65,6 +68,13 @@ private PreparedStatement prepareSelectExist(Session session) {
             .and(eq(ENQUEUE_ID, bindMarker(ENQUEUE_ID))));
     }
 
+    private PreparedStatement prepareDeleteOne(Session session) {
```

Comment:
aren't you afraid of tombstone?

Reply:
No, we don't care as we tombstone data that will never ever be read again :-)